vulns/readxl/RSEC-2023-0.yaml

```diff
@@ -0,0 +1,25 @@
+id: RSEC-2023-0
+details: An issue was discoverered in in function xls_getWorkSheet in xls.c in libxls 1.6.2, allows attackers to cause a
+ denial of service, via a crafted XLS file.
+affected:
+- package:
+    name: readxl
+    ecosystem: CRAN
+  ranges:
+  - type: ECOSYSTEM
+    events:
+    - introduced: 0.0.1
+    - fixed: 1.1.0
+  versions:
+  - 0.1.0
+  - 0.1.1
+  - 1.0.0
+references:
+- type: WEB
+  url: https://security-tracker.debian.org/tracker/source-package/r-cran-readxl
+- type: WEB
+  url: https://github.com/libxls/libxls/issues/94
```

Comment:
It might be helpful to point to the readxl issue tracker (https://github.com/tidyverse/readxl/issues/679). Since readxl uses a bundled version of libxls, mapping between the libxls upstream sources and the readxl published versions takes an extra step. IIRC the vulnerability was fixed by a cherry-pick in readxl.

Reply:
Okay, I've had a chance to review the reports and split this up accordingly (to my best knowledge). This is probably ready for a more thorough review if you're willing to take a look, and I'm happy to make any updates or add additional information.